src/coreclr/System.Private.CoreLib/src/System/RuntimeType.CoreCLR.cs

```diff
@@ -3896,6 +3896,23 @@ private void CreateInstanceCheckThis()
             return instance;
         }
 
+        /// <summary>
+        /// Creates an uninitialized object of the current type.
+        /// </summary>
+        /// <returns>An uninitalized instance of the object of the specified type.</returns>
+        [DebuggerStepThrough]
+        [DebuggerHidden]
+        internal object? CreateUninitializedInstance()
+        {
+            if (GenericCache is not ActivatorCache cache)
```

Comment:
The GenericCache is used for enum names and values already. If there is a code that works with enums a lot, we will likely see the two cache uses fight between each other.